scripts/prepare_signed_binaries.py

```diff
@@ -0,0 +1,102 @@
+# @file
+#
+# Copyright (c) Microsoft Corporation
+# SPDX-License-Identifier: BSD-2-Clause-Patent
+##
+"""A command line script to prepare the files generated from secure_boot_default_keys.py for a github release."""
+import argparse
+import logging
+import pathlib
+import shutil
+import sys
+import tempfile
+
+INSTRUCTIONS = r"""
+# Information
+
+Most users should depend on the Operating system to update the Secure Boot Signatures as otherwise the
+system will not boot. In a best case scenario, the system will boot but the user will be presented with
+a EFI_SECURIT_VIOLATION error message. Doing this out of band may effect cause security enforcement
```

Comment:
I think this section needs to be reworked. It's contradictory at the moment and the resulting behavior is not clear. It currently states:

- The system will not boot if Secure Boot signatures are updated.
- The system might boot but will reach a `EFI_SECURITY_VIOLATION` error.
- BitLocker recovery may be encountered.
- Advanced users can update their binaries.

A lot of this stems from too many assumptions made about system state without clearly explaining the assumptions. I would put the advanced user warning first and then associate each effect with a cause.